content/en/docs/admin/admission-controllers.md

```diff
@@ -21,7 +22,7 @@ is authenticated and authorized.  The controllers consist of the
 administrator. In that list, there are two special controllers:
 MutatingAdmissionWebhook and ValidatingAdmissionWebhook.  These execute the
 mutating and validating (respectively) [admission control
-webhooks](/docs/admin/extensible-admission-controllers/#external-admission-webhooks)
+webhooks](/docs/admin/extensible-admission-controllers.md#external-admission-webhooks)
```

Comment:
Update this link and all links that currently start with `/docs/` to the new structure, which starts with `/content/en/docs/`.